src/services/config.js

```diff
@@ -0,0 +1,93 @@
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const organizationExtensionsQueries = require('@database/queries/organizationExtensions')
+const { UniqueConstraintError, ForeignKeyConstraintError } = require('sequelize')
+const { Op } = require('sequelize')
+const responses = require('@helpers/responses')
+const _ = require('lodash')
+const { Organizations } = require('aws-sdk')
+
+module.exports = class configsHelper {
+	/**
+	 * List Configs.
+	 * @method
+	 * @name list
+	 * @param {Integer} organization_id
+	 * @param {String} id -  id.
+	 * @returns {JSON} - List of configs as response.
+	 */
+
+	static async list(organization_id) {
+		let orgExtenstionData = {}
+		let configData = []
+		try {
+			// define filter
+			const filter = {
+				organization_id,
+			}
+			// attributes to fetch from organisation Extenstion
+			const attributes = ['review_required', 'show_reviewer_list', 'min_approval', 'resource_type', 'review_type']
```

Comment:
@adithyadinesh0412 add all those values in env Variable file and also in env sample